api/app/services/validator/tags/img.rb

```diff
@@ -0,0 +1,12 @@
+module Validator
+  module Tags
+    # This class is a placeholder to demonstrate how we can intervene in the validation of
+    # a single HTML node and modify it in place during HTML validation. Currently, it does
+    # nothing except show the interface.
+    class Img
+      def validate_node!(node)
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `node`. If it's necessary, use `_` or `_node` as an argument name to indicate that it won't be used. You can also write as `validate_node!(*)` if you want the method to accept any arguments but don't care about them.